db/schema.rb

```diff
@@ -20,6 +20,7 @@
   create_table "users", force: :cascade do |t|
     t.string   "name",                   limit: 255,              null: false
     t.integer  "role",                   default: 0,  null: false
+    t.text     "profile_photo"
```

Comment:
Attachments are files, I think we should use image uploader ? it handles things like image resizing, and ensures the file you uploaded are images.